docs/en/setup/service-agent/java-agent/README.md

```diff
@@ -209,6 +213,7 @@ The tag is, key=`x-le` and value = `{"logic-span":true}`.
     * If you require customized metrics, try [SkyWalking Meter System Toolkit](Application-toolkit-meter.md).
     * If you want to print trace context(e.g. traceId) in your logs, choose the log frameworks, [log4j](Application-toolkit-log4j-1.x.md), 
 [log4j2](Application-toolkit-log4j-2.x.md), [logback](Application-toolkit-logback-1.x.md)
+    * If you want to report your logs by gRPC, choose the log frameworks, [log4j](Application-toolkit-log4j-1.x.md), [log4j2](Application-toolkit-log4j-2.x.md), [logback](Application-toolkit-logback-1.x.md)
```

Comment:
If you want to separate this, please separate the detail makrdown. It will be strange that both are pointing to `Application-toolkit-log4j-1.x.md`, etc.